pallets/market/src/lib.rs

```diff
@@ -524,6 +581,8 @@ pub mod pallet {
         DealDurationOutOfBounds,
         /// Deal's piece_cid is invalid.
         InvalidPieceCid,
+        /// Deal's parameters do no fall between what the SP has set with `publish_deal_parameters`
+        InvalidDealParameters,
```

Comment:
This is confusing on multiple levels.

* "Invalid parameters" would be closer to sending a string to a place expecting a number.
* The message seems targeted at who ever is pushing parameters, not "numbers being checked" for those parameters

I feel that something like this would be a bit more accurate:
```suggestion
        /// The proposed deal parameters' falls outside the parameter bounds set by the Storage Provider.
        OutOfBoundsDeal,
```